.nvmrc

```
@@ -1 +1 @@
-6.11.1
+16.17.1
```

```suggestion
lts/hydrogen
```

Je pense qu'on peut passer en Node 18 version LTS, qu'en penses-tu ?

---

Je suis pour :  je teste ça.

---

Testé et approuvé en Node 18 / npm 9 !